resources/howtos/search_bar_for_docs.md

```diff
@@ -131,8 +137,8 @@ module.exports = {
     [
       "vuepress-plugin-meilisearch",
       {
-        "hostUrl": "https://mymeilisearch.com",
-        "apiKey": "XXX",
+        "hostUrl": "<your-meilisearch-host-url>",
```

Comment:
I think there is the same kind of example to change in `For All Kinds of Documentation` section 😇